README.md

```diff
@@ -44,4 +46,31 @@ Then you can compile both `apache/james-project` and `linagora/openpaas-james` t
 mvn clean install -Dmaven.javadoc.skip=true
 ```
 
-You can add the `-DskipTests` flag as well if you don't want to run the tests of the `apache/james-project`.
\ No newline at end of file
+You can add the `-DskipTests` flag as well if you don't want to run the tests of the `apache/james-project`.
+
+### Building with a local jenkins runner
+
+You can use a custom local jenkins runner with the `Jenkinsfile` at the root of this project to build the project. 
+This will automatically do for you:
+
+* checkout and compile latest code of Apache James project alongside `openpaas-james`
+* generate docker images for `memory`, `distributed` and `distributed-ldap` flavors of the project
+* launch unit, integration and deployment tests on `openpaas-james`
+
+To launch it you need to have docker installed. From the root of this project, you can either use directly the image
+provided on docker hub:
+
+```
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/Jenkinsfile:/workspace/Jenkinsfile rcordier/jenkinsfile-runner:maven-jdk11
```

Comment:
let's not advertize this option and only ask people to build the jenkins-runner themselves

Reply:
Done :)